### DISCUSSION TITLE
Возможно ли скачать транскрибированный текст, а не аудио?

### DISCUSSION THREAD

**Discussion by Freaction** [ilyhalight/voice-over-translation#300] [Q&A] [answered]
Можно как-то скачать именно субтитры?

**Comment by NullVerdict** [comment]
> Данная функция была бы имбой, ибо нет нормального бесплатного сервиса, где можно было бы транскрибировать видео. А тут прям на тебе: и аудио можно скачать, и текст размеченный — идеалити.

Спасибо за бизнес идею.

**Comment by ilyhalight** [comment, marked as answer]
Если в меню есть прогруженные субтитры, то нужно зайти в консоль (devtools). После в консоли ищешь "[VOT] subtitles list" и оттуда скачиваешь субтитры на нужном языке. Открыть скачанный файл можно любым текстовым редактором. Формат субтитров JSON.
